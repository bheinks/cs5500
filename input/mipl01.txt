(*
	This file has a lexical error.
*)

program DiddleDeDum;
begin (* DiddleDeDum *)
  x := 1234567890123;
  write('e', 'n', 'd', '\')
end (*DiddleDeDum *) .
