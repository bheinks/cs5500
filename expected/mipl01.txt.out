TOKEN: T_PROG      LEXEME: program
N_PROGLBL -> T_PROG
TOKEN: T_IDENT     LEXEME: DiddleDeDum
TOKEN: T_SCOLON    LEXEME: ;
TOKEN: T_BEGIN     LEXEME: begin
N_VARDECPART -> epsilon
N_PROCDECPART -> epsilon
TOKEN: T_IDENT     LEXEME: x
TOKEN: T_ASSIGN    LEXEME: :=
N_VARIDENT -> T_IDENT
N_ENTIREVAR -> N_VARIDENT
N_VARIABLE -> N_ENTIREVAR
**** Invalid integer constant: 1234567890123
N_SIGN -> epsilon
Line 7: syntax error
